api/pkg/transformer/pipeline/operation.go

```diff
@@ -1,7 +1,118 @@
 package pipeline
 
-import "context"
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"sync"
+
+	"github.com/gojek/merlin/pkg/transformer/spec"
+	"github.com/gojek/merlin/pkg/transformer/types"
+	"github.com/gojek/merlin/pkg/transformer/types/table"
+)
 
 type Op interface {
```

Comment:
Can you add documentation for `Op` , `OperationIO`, `OperationTracing` ? including their field and methods. I think this is quite important part to be documented.